raiden/tests/scenarios/join-network-scenario-J7.yaml

```diff
@@ -0,0 +1,42 @@
+settings:
+  gas_price: "fast"
+
+token:
+
+nodes:
+  commands:
+    #start: "docker start raiden_{:02d}"
+    #stop: "docker stop raiden_{:02d}"
+    #kill: "docker kill raiden_{:02d}"
+    stop: "pkill -f '127.0.0.1:50{:02d}'"
+    kill: "pkill -9 -f '127.0.0.1:50{:02d}'"
+    start: "raiden --config-file raiden_{0:02d}.toml --api-address 127.0.0.1:50{0:02d}"
+  range:
+    first: 1
+    last: 4
+    template: "localhost:50{:02d}"
+
+scenario:
+  serial:
+    tasks:
+      - parallel:
+          name: "Setting up a network"
+          tasks:
+            - open_channel: {from: 0, to: 1, total_deposit: 10, expected_http_status: 201}
+            - open_channel: {from: 0, to: 2, total_deposit: 10, expected_http_status: 201}
+            - open_channel: {from: 1, to: 2, total_deposit: 10, expected_http_status: 201}
+
+      - serial:
+          name: "Checking the network"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 10, balance: 10, state: "opened"}
+            - assert: {from: 0, to: 2, total_deposit: 10, balance: 10, state: "opened"}
+            - assert: {from: 1, to: 2, total_deposit: 10, balance: 10, state: "opened"}
+
+      - serial:
+          name: "Node Nr. 4 joins with joinable_funds_target set to 0"
+          tasks:
+            - join_network: {from: 3, funds: 100, initial_channel_target: 3, joinable_funds_target: 0, expected_http_status: 204} # It opens with default values
```

Comment:
There is an issue for this, right? Can you link it here

Reply:
#2675